modules/tinymce/src/plugins/table/test/ts/browser/settings/TableDefaultAttributesWithColGroupsTest.ts

```diff
@@ -0,0 +1,130 @@
+import { ApproxStructure, GeneralSteps, Logger, Pipeline, Waiter } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { TinyApis, TinyLoader, TinyUi } from '@ephox/mcagar';
+import TablePlugin from 'tinymce/plugins/table/Plugin';
+import SilverTheme from 'tinymce/themes/silver/Theme';
+import * as TableTestUtils from '../../module/test/TableTestUtils';
+
+/* This requires a menubar. Cannot migrate yet. */
+UnitTest.asynctest('browser.tinymce.plugins.table.TableDefaultAttributesWithColGroupsTest', (success, failure) => {
+  TablePlugin();
+  SilverTheme();
+
+  TinyLoader.setup((editor, onSuccess, onFailure) => {
+    const tinyApis = TinyApis(editor);
+    const tinyUi = TinyUi(editor);
+
+    const createTableChildren = (s, str: ApproxStructure.StringApi, _arr) =>
```

Comment:
`_arr` isn't used here and is the reason for DeepScan failing, so I'd suggest just removing it.

Reply:
`s` could also be typed here I believe. eg: `s: ApproxStructure.StructApi`